src/features/quick-search/QuickSearchItem.js

```diff
@@ -159,10 +159,10 @@ const QuickSearchItem = ({
 const itemDataShape = {
     extension: PropTypes.string,
     iconType: PropTypes.string,
-    id: PropTypes.string.isRequired,
+    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
```

Comment:
item id's should always be strings

Reply:
typed ids should be strings. In EUA, I've seen it's number

Reply:
they should be strings or converted to strings from either the server or before it reaches this component

Reply:
ok, I've kept the type as string. I have an upcoming backend change so I'll ensure it's sending a string